tests/test_longitudinal.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import textwrap
+from longitudinal_sql import main
```

Comment:
rather than adding `sql/` to the python path, and importing `longitudinal_sql.main` here, i think this should use [`subprocess.check_output`](https://docs.python.org/3/library/subprocess.html#subprocess.check_output) and directly call `sql/longitudinal.sql.py`